### PR TITLE
Pranita_Assignment_9_Done

### PR DESCRIPTION
**Pull request by PranitaPJun20**
Design to test static, non static, return type concept.Added methods as per requirement.

### CODE REVIEW

File: src/pranitaPJun20/MethodCalling.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	double sum(double num1,double num2,double num3){
+		double sumLocal = num1+num2+num3;
+		//System.out.println("The sum is:"+sumLocal);
+		return sumLocal;	
+	}
+//Method average(), accept 3 parameters of type double & return answer
+	double average(double num1,double num2,double num3){
+		sum(80.5,90.9,78.8);
+		double averageLocal = (num1+num2+num3)/3;
+		//System.out.println("The average is:"+averageLocal);
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/MethodCalling.java]: line number 28, you should capture it in a variable and line number 30 should be.
double totalSum = sum(80.5,90.9,78.8);
double averageLocal = totalSum/3

File: src/pranitaPJun20/MethodCalling.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		return averageLocal;
+	}
+//Method isAnswerAboveExpecation(), take one parameter (answer which you received from average method)
+// and return true if value is more then 50 else false.
+	static boolean isAnswerAboveExpectation(){
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/MethodCalling.java]: You should pass argument here. You have already calculated average once, you should use that value in rest method by passing it as parameter.

File: src/pranitaPJun20/MethodCalling.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		double avg= methodCalling3.average(100,110,120);
+			if(avg>80)
+				return"A Grade";
+			else
+				return "B Grade";			
+		}
+		
+	public static void main(String[] args){
+		MethodCalling methodCalling = new MethodCalling();
+		methodCalling.average(100,90.99,78.89);
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/MethodCalling.java]: You should capture return value here and use it in isAnswerAboveExpectation() and isEligible() and getYourGrade() method

File: src/pranitaPJun20/MethodCalling.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/MethodCalling.java]: Done

File: src/pranitaPJun20/MethodCalling.java (hunk 2 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/MethodCalling.java]: Done

File: src/pranitaPJun20/MethodCalling.java (hunk 3 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/MethodCalling.java]: Done

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**